web/src/components/dashboard/SidebarCardFundingNudge.test.tsx

```diff
@@ -115,17 +227,49 @@ describe("<SidebarCardFundingNudge />", () => {
     });
   });
 
-  it("fires show_me_funding_opportunities analytics event when link is clicked", () => {
-    renderWithBusiness({
-      profileData: generateProfileData({
-        businessPersona: "STARTING",
-        industryId: "acupuncture",
-      }),
-      preferences: generatePreferences({ visibleSidebarCards: [fundingNudge] }),
+  describe("updates operating phase to UP_AND_RUNNING when missing information for tax filing", () => {
```

Comment:
Thanks for writing these tests. Glad to see we have coverage for 2 negative conditions, when the user doesn't provide a taxId and when they do not provide a businessName.

I think these are good tests, but we can improve on the naming. We want it to be immediately obvious what the purpose of the test is by just reading the names of the tests. The other 2 inner describe blocks are defining specific conditions:
- when clicking funding button for non-generic industry
- when clicking funding button for generic industry

And the tests inside describe specific intended outcomes:
- updates with new sector, marks tax registration task complete, and operating phase to UP_AND_RUNNING after modal success

So I can almost read this whole setup as a single sentence:
"when clicking funding button for generic industry, updates with new sector, marks tax registration task complete, and operating phase to UP_AND_RUNNING after modal success"

This clearly articulates what's happening and the desired outcome.